pages/add/form/creators.js

```diff
@@ -30,80 +35,17 @@ export default class Creators extends Component {
         console.error(reason);
       });
   }
-  updateCreators(creators) {
-    this.setState({ creators }, () => {
-      let relatedCreators = this.state.creators.map((creator) => {
-        // Model data thusly:
-        //
-        // "related_creators": [{
-        //   "creator_id": number | <id of the creator autocompleted. Will be null if the creator was not autocompleted and was manually typed in>,
-        //   "name": string | <name of the creator typed in, not autocompleted. Will be null if the creator was autocompleted>
-        // }, ...]
-        return {
-          creator_id: creator.id ? creator.id : null,
-          name: creator.id ? null : creator.name
-        };
-      });
-
-      this.props.onChange(null, relatedCreators);
-    });
-  }
-  handleDelete(i) {
-    const creators = this.state.creators.slice(0);
-    creators.splice(i, 1);
-    this.updateCreators(creators);
-  }
-  handleAddition(creator) {
-    creator.name = creator.name.trim();
-    const creators = [].concat(this.state.creators, this.fixCreatorLettercase(creator));
-    this.updateCreators(creators);
-  }
-  fixCreatorLettercase(creator) {
-    for (let t of this.state.suggestions) {
-      // We don't want to have duplicated creators in the database.
-      // If the user-defined creator is a known creator,
-      // fix the lettercase so it mataches the known creator we have in the database.
-      if (t.name.toLowerCase() === creator.name.toLowerCase()) {
-        creator = t;
-        break;
+  handlePendingInput(input) {
+    // If the creator the user typed matches a known creator suggestion (ignoring
```

Comment:
This isn't the behavior I get. Even when it's an exact match for the creator name, it gets discarded when I click away. (I should also point out that this differs from the delimiter case- even if there's exactly one match, we create a new creator unless the user specifically selects the match from the dropdown- Also worth creating a new ticket?)

Reply:
Turns out this happens because the profile marker is a bit of unicode string prepended to the user name.